### DISCUSSION TITLE
[A-L] (2023/24) Foglio 6 - Esercizio 5

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#176] [Esercizi - Bertini] [answered]
![Screenshot 2023-11-06 173905](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/9c46cbf4-e459-48c3-806e-52019e2e736a)

**Comment by Elia-Belli** [comment, marked as answer]
## Soluzione analoga a quella in classe
- **(1)** La probabilità che Alice superi l'esame è $\mathbb{P}(S)=\mathbb{P}(18/30)+...+\mathbb{P}(30/30)$, infatti l'evento di ottenere un determinato voto è disgiunto dall'evento di ottenerne un'altro.
Le risposte alle singole domande sono eventi indipendenti e siccome si risponde a caso la probabilità è uniforme sulle 4 possibili risposte, ovvero $p=1/4$, ne segue che:
$$\mathbb{P}(x\ corrette)=\binom{10}{x}p^x(1-p)^{10-x}$$
Notiamo però che dato il sistema di votazione, non è possibile ottenere tutti i voti ma solamente quelli dati dalla seguente formula $v=\\#corrette \cdot 3-\\#errate$, supponendo si possano ottenere voti negativi lo spazio dei possibilli voti è $\Omega=\\{4c-10|c=0,...,10\\}$ (formula ottenuta risolvendo il sistema con il vincolo $c+e=10$)
Per ottenere un voto sufficiente $4c-10\geq 18\Rightarrow c\geq 7$, ne segue che:
$$\mathbb{P}(S)=\sum^{10}_{c=7}\binom{10}{c}p^c(1-p)^{10-c}\simeq 0.035$$
- **(2)** Sia $X$ la v.a. che indica il voto ottenuto da Alice e $\bar{X}$ la v.a che indica il numero di domande corrette, è facile osservare che $\mathbb{P}(X=v)=\mathbb{P}(\bar X=c)$ con $v=4c-10$ perchè c'è una corrispondenza biunivoca tra voto e domande corrette.
Siccome $\bar X\sim Bin(10,1/4)$ sappiamo che $\mathbb{E}(\bar X)=np$, ma $\mathbb{E}$ è un'applicazione lineare, allora:
$$\mathbb{E}(X)=\mathbb{E}(4\bar X-10)=4\cdot \mathbb{E}(\bar X)-10=4\cdot \frac{10}{4} -10=0$$

Come verifica, utilizzando Wolfram Alpha otteniamo:

$$\mathbb{E}(X)=\sum_{v\in \Omega}v\cdot \mathbb{P}(X=v)=0$$

$$\mathbb{V}(X)=\sum_{v\in \Omega}v^2\cdot \mathbb{P}(X=v)=30$$

<p align="center">
<img width=300 src=https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/e0d27b35-293c-4cf8-b8dc-52a98759b47d>
</p>

**Comment by CiottoloMaggico** [comment]
![immagine](https://github.com/user-attachments/assets/1be599a2-6914-4b35-b408-e92f0b2850f7)

**Comment by Elia-Belli** [comment]
## Soluzione del prof
- **(2)** Siano $X_g,X_s$ rispettivamente le v.a. che indicano le domande giuste e sbagliate ottenute:
$X=3\cdot X_g-X_s=3X_g-(10-X_g)=4\cdot X_g-10$
Sappiamo che $X_g$ ha distribuzione binomiale quindi $\mathbb{E}(X_g)=np=10/4$ allora $\mathbb{E}(X)=4\cdot 10/4-10=0$
- **(3)** Siccome $\mathbb{V}(X)=\mathbb{V}(4\cdot X_g-10)=16\cdot \mathbb{V}(X_g)=16\cdot (10/4 \cdot 3/4)=30 $
